lib/apress/selenium_products/spec/company_site/product_creation_page/new_form_spec.rb

```diff
@@ -0,0 +1,621 @@
+          it 'на КТ отобразится цена со скидкой и старая цена' do
+            expect(@product_page.product_price).to include '300'
+            expect(@product_page.product_price).to include '500'
+          end
+        end
+      end
+    end
+
+    describe 'до отправки формы' do
+      describe 'Рубрика' do
+        let(:options) { {rubric: CONFIG['rubrics_names_by_level'][5]} }
+
+        context 'когда привязывается через выбор в дереве рубрик' do
+          before do
+            @product_creation_page.rubric_level1
+            @product_creation_page.rubric_level2
+            @product_creation_page.rubric_level3
+            @product_creation_page.rubric_level4
+            @product_creation_page.rubric_level5
+          end
+
+          it 'отобразится превью привязки к рубрике' do
+            expect(@product_creation_page.check_rubric).to be_truthy
+          end
+        end
+
+        context 'когда привязывается через поиск' do
+          before { @product_creation_page.fill_rubric(options) }
+
+          it 'отобразится превью привязки к рубрике' do
+            expect(@product_creation_page.check_rubric).to be_truthy
+          end
+        end
+
+        context 'когда перевыбираем рубрику' do
+          before do
+            @product_creation_page.fill_rubric(options)
+            options[:rubric] = CONFIG['rubric_2'] # переустанока в переменную другой рубрики
+            @product_creation_page.fill_rubric(options)
+          end
+
+          it 'отобразится вторая рубрика' do
+            expect(@product_creation_page.check_rubric).to include(options[:rubric])
+          end
+        end
+      end
+
+      describe 'Краткое описание' do
+        context 'когда введено < 50 символов' do
+          let(:options) { {short_description: CONFIG['product_creation']['short_description']['less_than_50']} }
+
+          before { @product_creation_page.short_description = options[:short_description] }
+
+          it 'под полем КО выводится предупреждение, что товар будет без КО, т.к. < 50 символов' do
+            expect(@product_creation_page.short_description_error).to include '- менее 50 символов'
+          end
+        end
+
+        context 'когда текст из КО совпадает с Названием' do
+          let(:options) do {
+            name: CONFIG['product_creation']['name']['valid'],
+            short_description: CONFIG['product_creation']['short_description']['as_the_name'] }
+          end
+
+          before do
+            @product_creation_page.name_input = options[:name]
+            @product_creation_page.short_description = options[:short_description]
+          end
+
+          it 'под полем КО выводится предупреждение, что товар будет без КО, т.к. слова из названия' do
+            expect(@product_creation_page.short_description_error).to include '- слова из названия'
+          end
+        end
+
+        context 'когда введено > 50 символов и не совпадает с названием' do
+          let(:options) do {
```

Comment:
[Correctable] Layout/MultilineBlockLayout: Block body expression is on the same line as the block start.